hack/scripts/import-crds.sh

```diff
@@ -95,5 +95,5 @@ crd-importer \
     --out=./charts/kubedb-grafana-dashboards/crds
 
 crd-importer \
-    --input=https://github.com/kubeops/supervisor/raw/v0.0.1/crds/supervisor.appscode.com_recommendations.yaml \
+    --input=https://github.com/kubeops/supervisor/raw/v0.0.2/crds/supervisor.appscode.com_recommendations.yaml \
```

Comment:
Should not we import the scanner crds with `--input=https://github.com/kubeops/scanner/tree/v0.0.5/crds/<yamls>`?

Reply:
Currently not as the crds in v0.0.5 are not same as status branch crds!